src/client/components/app/LineDetailsDisplay/LineMetadata.tsx

```diff
@@ -302,12 +356,12 @@ export const LineMetadata = () => {
           actions={[
             {
               name: t('components.transaction.deposit'),
-              handler: () => depositHandler(token),
+              handler: () => addCollateralHandler(token),
```

Comment:
thank you for changing the names - I left a TODO because the incorrect naming was bothering me